packages/vanilla-bundle/src/components/__tests__/slick-footer.spec.ts

```diff
@@ -85,7 +86,7 @@ describe('Slick-Footer Component', () => {
 
       const footerContainerElm = document.querySelector('div.slick-custom-footer.slickgrid_123456') as HTMLDivElement;
       const leftFooterElm = document.querySelector('div.slick-custom-footer.slickgrid_123456 > div.left-footer') as HTMLSpanElement;
-      const rightFooterElm = document.querySelector('div.slick-custom-footer.slickgrid_123456 > div.metrics') as HTMLSpanElement;
+      const rightFooterElm = document.querySelector('div.slick-custom-footer.slickgrid_123456 > div.right-footer') as HTMLSpanElement;
```

Comment:
Should the lookup be 'div.right-footer.metrics'?

Reply:
no not in this case because the `metrics` css class name is only added if it's not defined as hidden, that was refactored in this PR is different compare to previous code.

Since this unit test is to hide it, it won't add css class because of this code
```ts
// ...
else if (!this.customFooterOptions.hideMetrics) {
      rightFooterElm.classList.add('metrics');
```

Reply:
Thank you for the clarifications. It makes senses now.